### PR TITLE
ci: tell Angular CLI not to update webdriver

### PR DESCRIPTION
**Pull request by alexeagle**
It can get ahead of the Chrome version on our CI machine

### CODE REVIEW

File: examples/angular_bazel_architect/package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "test": "bazel test :test",
     "lint": "bazel run :lint",
     "e2e": "bazel test :e2e",
-    "postinstall": "patch-package && ngcc --properties es2015 browser module main --first-only"
+    "postinstall": "patch-package && ngcc --properties es2015 browser module main --first-only",
+    "webdriver-update": "webdriver-manager update --standalone false --gecko false --versions.chrome 85.0.4183.38"
```
Comment (alan-agius4) [inline examples/angular_bazel_architect/package.json:12]: Shouldn't you call `webdriver-update` post-install?

Comment (alexeagle) [reply examples/angular_bazel_architect/package.json:12]: thx, added `&& yarn webdriver-update` after patch-package

**Review by alan-agius4 — APPROVED** [review]

File: examples/angular_bazel_architect/BUILD.bazel [hunk 2]
```diff
@@ -102,7 +102,10 @@ architect_test(
 
 architect_test(
     name = "e2e",
-    args = ["frontend:e2e"],
+    args = [
+        "frontend:e2e",
+        "--no-webdriver-update",
```
Comment (alan-agius4) [inline examples/angular_bazel_architect/BUILD.bazel:107]: The architect CLI, doesn't transform kebabed cases args to camel. But the schema validation needs camel cased properties.

```suggestion
        "--webdriverUpdate=false",
```

**Review by alan-agius4 — APPROVED** [review]
LGTM, after the arg change

File: examples/angular_bazel_architect/BUILD.bazel (hunk 2 above)
Comment (alexeagle) [reply examples/angular_bazel_architect/BUILD.bazel]: hmm still fails
```
Error: Schema validation failed with the following errors:
  Data path ".webdriverUpdate" should be boolean.
```

Comment (alan-agius4) [reply examples/angular_bazel_architect/BUILD.bazel]: I looked at this and I pushed a commit (I hope you don't mind) which should address the problem.

**Pull request merged**